### PR TITLE
[USER32][NTUSER] Fill by white in DrawFrameControl:DFC_MENU

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

`DrawFrameControl:DFC_MENU` draws the monochrome image of menu arrow or checkmark. However, the function didn't draw the entire rectangle correctly when the rectangle was not a square.
`DrawFrameControl:DFC_MENU` should fill the background rectangle by white.

JIRA issue: [CORE-18417](https://jira.reactos.org/browse/CORE-18417)

## Comparison

BEFORE:
before

AFTER:
after

## TODO

- [x] Do tests.

### CODE REVIEW

File: win32ss/user/user32/windows/draw.c [hunk 1]
```diff
@@ -978,6 +978,7 @@ static BOOL UITOOLS95_DrawFrameMenu(HDC dc, LPRECT r, UINT uFlags)
     TCHAR Symbol;
     RECT myr;
     INT cxy;
+    FillRect(dc, r, (HBRUSH)GetStockObject(WHITE_BRUSH));
```
Comment (learn-more) [inline win32ss/user/user32/windows/draw.c:981]: Are you sure `WHITE_BRUSH` is the correct brush?

**Review by learn-more — COMMENTED** [review]
There is also an `UITOOLS95_DrawFrameMenu` in `win32ss/user/ntuser/draw.c`, this probably also needs fixing?

File: win32ss/user/user32/windows/draw.c (hunk 1 above)
Comment (katahiromz) [reply win32ss/user/user32/windows/draw.c:981]: Yes, must be white. I confirmed.

**Review by learn-more — APPROVED** [review]

**Comment by learn-more** [comment]
An optimization would be to not draw this unless it is required, or disable this drawing operation while drawing the arrow (set background mode to transparent)

**Pull request merged**